app/helpers/brexit_checker_helper.rb

```diff
@@ -8,14 +8,16 @@ def format_criteria_list(criteria)
   def format_action_audiences(actions)
     action_groups = actions.group_by(&:audience)
 
-    action_groups.map do |key, action_group|
+    action_groups = action_groups.map do |key, action_group|
```

Comment:
It's hard to tell from reading this why we're reversing the groups. Can we do a `sort_by` here, so the ordering predicate is clear?

Reply:
Went through with Jos and simplified massively. @benthorner hopefully the result is much easier to grok?